pavement.py

```diff
@@ -6,7 +6,7 @@
 NAME = 'blackbelt'
 VERSION = '0.12.1'
 
-requires = ['click', 'requests', 'PyGithub', 'slacker', 'pyopenssl', 'ndg-httpsclient', 'pyasn1']
+requires = ['click', 'requests', 'PyGithub', 'slacker==0.6.2', 'pyopenssl', 'ndg-httpsclient', 'pyasn1']
```

Comment:
For now, I add minimal version `Slacker` to both places and in some future PR we can solve this problem.